### DISCUSSION TITLE
Clarification needed around save game sync options - upload, download, force download, force upload

### DISCUSSION THREAD

**Discussion by msmirle** [Heroic-Games-Launcher/HeroicGamesLauncher#1739] [Q&A] [answered]
Hi there,

I've been trying to find documentation around the different save functions in HGL. I've found some explanations but a lot of answers on Reddit are poorly worded and conflicting.

Can someone confirm that the below is accurate? This is the most clear explanation that I've found:

- **Download** - attempt to download your cloud save to your device. this will replace your device save if the cloud is newer.
- **Upload** - attempt to upload your device save to the cloud. this will replace your cloud save if the device is newer.
- **Force Download** - download your cloud save to your device. this will overwrite your device save even if it is newer.
- **Force Upload** - upload your device save to the cloud. this will overwrite your cloud save even if it is newer.

**Comment by imLinguin** [comment, marked as answer]
Yes, this is accurate.

**Reply by msmirle** [reply]
Thanks - much appreciated. Is this actually documented anywhere? I know it seems honest but honestly there are people confused out there :)

**Comment by flavioislima** [comment]
We should add this help text to the screen, maybe using a help icon or help box.
